### ISSUE TITLE
Request translations

### ISSUE THREAD

**Issue by amoraitis** [nisrulz/android-tips-tricks#12]
Request translations type:README-languagecode.me from others.
Btw I am in to start creating the readme for the Greek language

**Comment by nisrulz** [comment]
Good idea. I will put the information in the current readme and the instructions to translate. If you would want to translate this into Greek, go ahead and send a PR when you are done. The format would be same with file name being `README-languagecode.md`.

Thank you for your suggestion 😄

**Comment by nisrulz** [comment]
@amoraitis do you think you are still up for translating this into greek?

Let me know if you can. 

Thank you

**Comment by nisrulz** [comment]
I have added the translation information in the readme. Translations for this repo are all welcome.

Translation already in works:
Greek -  by @amoraitis

**Comment by amoraitis** [comment]
OK, I am working too slowly in this, but I will end it soon.

**Comment by Andreas-White** [comment]
@amoraitis if there is a need for help, also a native Greek (αμα χρειαζεσαι βοηθεια...).

**Comment by amoraitis** [comment]
> @amoraitis if there is a need for help, also a native Greek (αμα χρειαζεσαι βοηθεια...).

Feel free to take over. I am no longer working on this. Look at my fork.

**Comment by nisrulz** [comment]
As a side note, this issue is very old and at that time it wasn't quite easy to support such a huge document. As of current state this repo uses `mdBook` and has also split the content in many places. 

mdBook doesn't "yet" support multi-lingual book: https://github.com/rust-lang/mdBook/issues/5

So my suggestion would be translate everything inside [src](https://github.com/nisrulz/android-tips-tricks/tree/master/src) directory, one file at a time and drop them into another `src-gr` directory at the root of the repository

![Screenshot 2021-03-05 at 16 35 38](https://user-images.githubusercontent.com/2096087/110137181-d844f700-7dd0-11eb-9c3f-cbd15d0d9724.png)

What I can do is create a versions of the book for each language type and then link them together.
